src/modules/core/components/Popover/PopoverSection/ColonySection.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { DropdownMenuItem, DropdownMenuSection } from '~core/DropdownMenu';
+import NavLink from '~core/NavLink';
+import { CREATE_COLONY_ROUTE } from '~routes/routeConstants';
+
+const MSG = defineMessages({
+  createColony: {
+    id: 'users.AvatarDropdown.AvatarDropdownPopover.link.createColony',
+    defaultMessage: 'Create a Colony',
+  },
+});
+
+const displayName = 'users.AvatarDropdown.AvatarDropdownPopover.ColonySection';
+
+const ColonySection = ({ itemClassName }: { itemClassName?: string }) => (
```

Comment:
This is not a core component, it cannot be shared with others. It's a `user` level component